frontend/layout/footer/Footer.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+import styled from "styled-components"
+
+/**Section */
+const FooterSection = styled.section `
+
+    width: 100%;
+    height: 200px;
+    background-color: #F1F1F1;
+    display: flex;
+    @media (max-width: 670px) {
+        height: 400px;
+        flex-direction: row;
+    }
+`
+const Container  = styled.div `
+
+    width: 500px;
+    height: 200px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    @media (max-width: 670px) {
+        width: 250px;
+        height: 400px;
+        justify-content: flex-end;
+    }
+`
+const ResponsiveContainer = styled.div `
+
+margin-left:10%;
+    display: flex;
+    @media (max-width: 670px) {
+        height: 400px;
+        flex-direction: column;
+        margin-left:0%;
+    }
+    
+`
+/**Fonts */
+const FooterH3 = styled.h3 `
```

Comment:
všechny "há" elementy(h1, h2,...) bych v moderním vývoji moc nepoužíval, lepší je div nebo span nastylovanej podle globálních pravidel v layout/Theme.ts...há trojky mají tendenci se za určitých okolností chovat na více prohlížečích různě(třeba v Chromu to jede v pohodě, ale v Mozille máš font defaultní, apod...)